content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -319,4 +319,46 @@ There are at least two workarounds:
 ```bash
 kubectl taint nodes NODE_NAME role.kubernetes.io/master:NoSchedule-
 ```
+
+## A Linux distribution has a read-only /usr directory
```

Comment:
"When the Linux distribution for a node has a read-only mounted `/usr` directory"

Reply:
@rosti updated.